applications/rag/README.md

```diff
@@ -1,192 +1,204 @@
-# RAG-on-GKE Application
+# RAG on GKE
 
-**NOTE:** This solution is in beta. Please expect friction while using it.
+## What is RAG?
 
-This is a sample to deploy a RAG application on GKE. Retrieval Augmented Generation (RAG) is a popular approach for boosting the accuracy of LLM responses, particularly for domain specific or private data sets. The basic idea is to have a semantically searchable knowledge base (often using vector search), which is used to retrieve relevant snippets for a given prompt to provide additional context to the LLM. Augmenting the knowledge base with additional data is typically cheaper than fine tuning and is more scalable when incorporating current events and other rapidly changing data spaces.
+This is a sample to deploy a Retrieval Augmented Generation (RAG) application on GKE. RAG is a popular approach for boosting the accuracy of LLM responses, particularly for domain specific or private data sets.
 
-Architecture:
-1. A k8s service serving Hugging Face TGI inference using `mistral-7b`.
-2. Cloud SQL `pgvector` instance with vector embeddings generated from the input dataset.
-3. A k8s front end chat interface to interact with the inference server and the vector DB.
-4. Ray cluster runs job to populate vector DB.
-5. Jupyter notebook to read dataset and trigger ray job to populate the vector DB.
+RAG uses a semantically searchable knowledge base (like vector search) to retrieve relevant snippets for a given prompt to provide additional context to the LLM. Augmenting the knowledge base with additional data is typically cheaper than fine tuning and is more scalable when incorporating current events and other rapidly changing data spaces.
 
-## Installation
+#### RAG on GKE Architecture
+1. A GKE service endpoint serving [Hugging Face TGI inference](https://huggingface.co/docs/text-generation-inference/en/index) using `mistral-7b`.
+2. [Cloud SQL `pgvector` instance](https://github.com/pgvector/pgvector) with vector embeddings generated from an input dataset.
+3. A [Ray](https://docs.ray.io/en/latest/ray-overview/getting-started.html) cluster running on GKE that runs jobs to generate embeddings and populate the vector DB.
+5. A [Jupyter](https://docs.jupyter.org/en/latest/) notebook running on GKE that reads the dataset using GCS fuse driver integrations and runs a Ray job to populate the vector DB.
+3. A front end chat interface running on GKE that prompts the inference server with context from the vector DB.
+
+The tutorial walks you through installing the RAG infrastructure in a GCP project, generating vector embeddings for a sample [Kaggle Netflix shows](https://www.kaggle.com/datasets/shivamb/netflix-shows) dataset and prompting the LLM with context.
```

Comment:
Can you update the PG vector table name to reflect the dataset: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/example_notebooks/rag-kaggle-ray-sql-latest.ipynb?short_path=998dd30#L125

Reply:
This is done in this PR already